### PR TITLE
Franz to Authors :)

### PR DESCRIPTION
**Pull request by ax3l**
Move Franz to authors for the next release!

MPI benchmark and JSON in-coming :) Thank you! :rocket: :sparkles:

### CODE REVIEW

File: .rodare.json [hunk 1]
```diff
@@ -5,6 +5,10 @@
             "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
             "orcid": "0000-0001-8704-1769"
         },
+        {
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "name": "Poeschel, Franz"
```
Comment (ax3l) [inline .rodare.json]: @franzpoeschel if you have or want to create an [orcid](https://orcid.org/) for your scientific career, now could be a good time :)

Comment (franzpoeschel) [reply .rodare.json]: Ah right, wanted to create one and forgot about it.
`0000-0001-7042-5088`

Comment (ax3l) [reply .rodare.json]: Cool, added! :)
You can also add it to your HZDR profile here: https://www.hzdr.de/db/UserDB.UserWWW.Run

File: .rodare.json [hunk 2]
```diff
@@ -5,6 +5,11 @@
             "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
             "orcid": "0000-0001-8704-1769"
         },
+        {
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "name": "Poeschel, Franz",
```
Comment (ax3l) [inline .rodare.json]: Do you want to be spelled "oe" or ö? :)

File: .rodare.json (hunk 1 above)
Comment (ax3l) [reply .rodare.json]: Do you want to be spelled "oe" or ö? :)

Comment (ax3l) [reply .rodare.json]: @franzpoeschel do you want to be spelled "oe" or "ö"? :)

**Pull request merged**